docs/docs/services/qix-engine/access-control.md

```diff
@@ -218,13 +330,13 @@ This operator inverts the Boolean value of an expression and
 returns `true` if the expression is `false` and returns `false`
 if the expression is `true`.
 
-**Syntax**
+##### Syntax
```

Comment:
Defining these as headers will cause the table of contents to become extremely repetitive

Reply:
:+1: Will revert